nevis/_util.py

```diff
@@ -110,3 +112,141 @@ def print_result(x, y, z):
     photo = hill.photo()
     if photo:
         print('  ' + photo)
+
+
+def generate_kml(file, labels=None, trajectory=None, points=None,):
+    """
+    Generate a KML (Keyhole Markup Language, used to display geographic data
+    in an Earth browser such as Google Earth) file from given data.
+
```

Comment:
Please add a 2nd paragraph with something like "For a description of the KML format, see ..."